benchmarks/frameworks/torch/mpc/pytest-benchmark/bench.py

```diff
@@ -0,0 +1,25 @@
+"""
+Define benchmark tests
+"""
+
+from workers_initialization import workers, hook
+from benchmark_functions import sigmoid, tanh
+
+# Initialize workers globally for the tests
+worker = workers(hook())
```

Comment:
since we are using pytest, we can create a conftest.py and can have workers as fixtures there :)